hummingbird/templates/base.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+{% load staticfiles %}
+<script src="{% static "js/jquery.js" %}"></script>
+<script src="{% static "js/hummingbird-ajax.js" %}"></script>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="icon" href="http://getbootstrap.com/favicon.ico">
```

Comment:
I don't have one, but I'm fine with not having any favicon if we don't have anything. I just don't think we should have the Bootstrap favicon. :P